### DISCUSSION TITLE
Import without tagging - avoid duplicates

### DISCUSSION THREAD

**Discussion by M1scer** [beetbox/beets#4813] [Q&A]
Hi all,

is there a way to Import music without tagging but avoid duplicates in case of importing a file twice?

Background:
I am using spotdl to download and tag music. But those files are downloaden into one folder and I can not avoid downloading a file twice.

**Comment by arsaboo** [comment]
You can use `duplicate_action` - https://beets.readthedocs.io/en/latest/reference/config.html#duplicate-action

**Reply by M1scer** [reply]
I guess this only workes if the Auto tagging function is active? My files are already tagged, i do an aß-is Import.

**Comment by sampsyo** [comment]
One option would be to import normally (without autotagging) and then use the `duplicates` plugin to detect duplicates after the fact.

**Comment by jpfieber** [comment]
I'm struggling with the same thing. I do my tagging in Picard (prefer the interface), then import as-is. With duplicate_action set to ask, I am not prompted, the duplicate files are just imported into my library.  This seems like it should be such a base level task for a music library system, is there really no way to prevent duplicates from being imported? The Duplicates plugin is a clean-up-after-the-fact option, I don't want the duplicates in the first place, so it's not useful.

**Reply by NicolasPL64** [reply]
Any solution yet? Going through the same issue here, super annoying

**Reply by snejus** [reply]
Which beets version are you using? This should have been fixed in [2.7.0](https://beets.readthedocs.io/en/latest/changelog.html#id11).

**Reply by NicolasPL64** [reply]
Latest.
It seems like I messed up by trying to reimport the whole library using the destination folder as the input.
Had to fix it manually.

**Reply by snejus** [reply]
OK, I see. Can you confirm whether duplicate detection works for asis imports? And if not, would you mind submitting an issue?
